style.css

```diff
@@ -141,7 +231,203 @@ body {
   margin-left: 49%;
 }
 
-.rece {
+.title-1 {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  margin-left: -5%;
+}
+
+/* popup mobile */
+
+.main {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  background: #c1c7d0;
+}
+
+.maincard {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  width: 90%;
+  background: #fff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  z-index: 1;
+  height: 96%;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 20px;
+  margin-bottom: 26px;
```

Comment:
Kindly fix the issue mentioned by the previous reviewers as your modal still have the buttom empty space this is caused by fixed height , also please use percentages and flexbox to align your items  rather than pixels for better responsiveness